lib/ansible/plugins/lookup/skydive.py

```diff
@@ -0,0 +1,77 @@
+#
+#  Copyright 2018 Red Hat | Ansible
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+lookup: skydive
+version_added: "2.8"
+short_description: Query Skydive objects
+description:
+  - Uses the Skydive python REST client to return the queried object from
+    Skydive network analyser.
+requirements:
+  - skydive-client
+extends_documentation_fragment: skydive
+options:
+    filter:
+      description: a dict object that is used to filter the return objects
+"""
+
+EXAMPLES = """
+- name: return skydive metdata if present based on Name
+  set_fact:
+    skydive_meta: "{{ lookup('skydive_lookup', filter={'Name':'test-VirtualBox'}) }}"
+
+- name: return skydive metdata if present based on TID
+  set_fact:
+    skydive: "{{ lookup('skydive_lookup', filter={'TID': '2b5e8263-89d3-5e01-506b-9120f49572b5'},
```

Comment:
I tried to use/test it without success. I get the following error:

```
fatal: [localhost]: FAILED! => {"msg": "lookup plugin (skydive_lookup) not found"}
```

Is there a good (doc ?) way to deploy it properly so that we could test it and maybe create a CI job.